tests_config/run_spec.js

```diff
@@ -14,12 +14,12 @@ const ALL_PARSERS = process.env["ALL_PARSERS"]
 
 function run_spec(dirname, options, additionalParsers) {
   fs.readdirSync(dirname).forEach(filename => {
-    const extension = extname(filename);
+    const path = dirname + "/" + filename;
     if (
-      /^\.([jt]sx?|css|graphql|html)$/.test(extension) &&
```

Comment:
Replaced this whitelist regex with a blacklist so we don't have to keeping adding file extensions to it!